bin/node-lambda

```diff
@@ -21,7 +22,7 @@ var AWS_HANDLER = process.env.AWS_HANDLER || 'index.handler';
 var AWS_ROLE = process.env.AWS_ROLE_ARN || process.env.AWS_ROLE || 'missing';
 var AWS_MEMORY_SIZE = process.env.AWS_MEMORY_SIZE || 128;
 var AWS_TIMEOUT = process.env.AWS_TIMEOUT || 60;
-var AWS_DESCRIPTION = process.env.AWS_DESCRIPTION || '';
+var AWS_DESCRIPTION = process.env.AWS_DESCRIPTION || packageJson.description || '';
```

Comment:
`description` is not a required `package.json` field, so we might want to check if `description` is in `packageJson` before trying to load it